var/spack/repos/builtin/packages/bdsim/package.py

```diff
@@ -32,3 +34,10 @@ class Bdsim(CMakePackage):
     depends_on("clhep")
     depends_on("flex")
     depends_on("bison")
+
+    patch("c++-standard.patch", when="@:1.7.6")
+
+    def cmake_args(self):
+        args = []
+        args.append("-DCMAKE_CXX_STANDARD=%s" % self.spec["root"].variants["cxxstd"].value)
```

Comment:
Optional:  We are recommending use of f-strings (https://docs.python.org/3/tutorial/inputoutput.html), which we are slowly converting to in Spack core.

Reply:
Changed!